delete-me.rb

```diff
@@ -0,0 +1,3 @@
+require 'google-translate'
```

Comment:
pls remove this file, and be more careful when you add files... *always* run `git status` and `git diff` before you add files!